### ISSUE TITLE
WebTorrent won't download from Magnet Link or Torrent address - Win 10

### ISSUE THREAD

**Issue by sbb1703** [webtorrent/webtorrent-desktop#1793]
Hi,  I'm new to Github so excuse any error in use.

My WebTorrent today stopped accepting torrents by any method - Magnet Link, normal download or address. There is simply no response. It was working fine and then nothing.

Using Windows 10 Desktop ver. 1903 for x64

Anyone help. It is probably something really simple

Thanks Sean

**Comment by brbergami** [comment]
Wow, I came here looking for a confirmation if I'm a huge idiot that missed something at some point, but I'm glad to see the same issue that I have.. Win10 ProN 1909, WT 0.21.0, first time using the app as seeder. After creating folder torrents (as non private), they wont seed. I have 2 computers under the same network (but I'm completely aware that I'm using AP isolation for 2 VAP in the router, so the downloads must "leave LAN" to actually communicate in P2P network as I tested with applications like Resilio). Both computers can download and seed the default videos that the app comes with. But I can't create my own. In advanced torrent creation I can see the tracker list it's ok. So no clue why the computers can't see eachother.
